### PR TITLE
Feat/edge radius

### PR DESCRIPTION
**Pull request by eactisgrosso**
Hi @shahnawaz,

There you go, the edge radius.
Let me know if you need me to modify anything.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -40,6 +40,7 @@ const styles = StyleSheet.create({
     height: '100%',
     alignItems: 'center',
     justifyContent: 'space-around',
+    zIndex: -1
```
Comment (shahnawaz) [inline index.js:43]: Just to understand, why did you have to add `zIndex`? (a screenshot of what was not right without it?)

**Comment by shahnawaz** [comment]
Hey @eactisgrosso 

Looks good to me.

Can you attach some screenshots (here) of how it looks now for reference?

File: index.js (hunk 1 above)
Comment (eactisgrosso) [reply index.js:43]: Sure thing:
- Without zIndex:
IMG_2711

- With zIndex:
IMG_2713

Comment (shahnawaz) [reply index.js:43]: ah.. nice

Comment (eactisgrosso) [reply index.js:43]: We have been using it for a while like that in Android and iOS and it has been working fine, but my team upgraded my local version to react hooks, so it took me a while to make the diff and test it.

Comment (shahnawaz) [reply index.js:43]: @electhuang 

oh, right. We also have some pending items for this package, if you are willing, feel free to pick any of them.

Comment (eactisgrosso) [reply index.js:43]: Let me ask @chescalante from my team, he already updated the component to hooks.

Comment (shahnawaz) [reply index.js:43]: Thanks, that would be awesome.

**Pull request merged**